### ISSUE TITLE
Server suddenly stopped working

### ISSUE THREAD

**Issue by KAW0** [FubarDevelopment/WebDavServer#50]
1. I start the server and put some files there. (I tested WinSCP and GoodSync)
2. Sending files is successful. If it writes files to SQLite they appear correctly in the database, but if it changes to DOTNET they do not appear in the file system.
3. Regardless of the method used to save the files, an attempt to read the files or send more will cause the server to stop suddenly with this messege:

`Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker:Information: Executed action method FubarDev.WebDavServer.Sample.AspNetCore.Controllers.WebDavController.GetAsync (FubarDev.WebDavServer.Sample.AspNetCore), returned result FubarDev.WebDavServer.AspNetCore.WebDavIndirectResult in 10.6713ms.
Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker:Trace: Action Filter: Before executing OnActionExecuted on filter Microsoft.AspNetCore.Mvc.ModelBinding.UnsupportedContentTypeFilter.
Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker:Trace: Action Filter: After executing OnActionExecuted on filter Microsoft.AspNetCore.Mvc.ModelBinding.UnsupportedContentTypeFilter.
Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker:Trace: Before executing action result FubarDev.WebDavServer.AspNetCore.WebDavIndirectResult.
Microsoft.AspNetCore.Mvc.StatusCodeResult:Information: Executing HttpStatusCodeResult, setting HTTP status code 200
The program „[17424] dotnet.exe” has exited with code -1073740940 (0xc0000374).`

Previously I thought that this is the reason:
[#49](https://github.com/FubarDevelopment/WebDavServer/issues/49)
but if it is normal behaviour I have no idea what is wrong.
I use Netcore 2.1 on Windows 10. Project run without IIS express.

**Comment by fubar-coder** [comment]
This looks like a windows error code: `c0000374` and means that there is a problem with an invalid pointer. To be honest, I don't have an idea why and where this happens, but I suggest that you try updating to .NET Core 2.2.

The `UnsupportedContentTypeFilter` may be something that I can fix though. Do you see this error message when you use WinSCP?

**Comment by KAW0** [comment]
Nothing changed .NET Core 2.2
WinSCP Error Messege: `Could not read status line:  An existing connection was forcibly closed by the remote host`

**Comment by fubar-coder** [comment]
What's the version of the WebDAV server nuget packages you used?

What happens if you use the NuGet packages from MyGet?

Feed URL: https://www.myget.org/F/webdav-server/api/v3/index.json
MyGet gallery page: https://www.myget.org/gallery/webdav-server

Does the problem still exist when you use IIS express?
Does this problem occur on a different computer? (Just publish, copy, start)

This is really strange, because there are definitely other users of this library and I guess that this problem is on your side.

**Comment by KAW0** [comment]
I'm really no kidding but it's seems to that publishing fixing this issue.  I published the project and launched it without VS and everything work correctly. After that i back to VS and tested this again and everything was fine... I unpack a11219e release in another directory and opened it as another project and the problem returned. I publish this and... it was fixed again...

IIE Express is another story it does not work at all:
`HTTP Error 502.5 - Process Failure`
